Source/tess.c

```diff
@@ -938,7 +938,11 @@ void tessAddContour( TESStesselator *tess, int size, const void* vertices,
 	{
 		const TESSreal* coords = (const TESSreal*)src;
 		src += stride;
-
+		if (isnan(coords[0]) || isnan(coords[1]) || (size > 2 && isnan(coords[2]))) {
+			// "Out of memory" isn't quite right, but give up and bail out
```

Comment:
That `outOfMemory` seems to be used as a general error. I would recommend to rename it to `invalidInput` instead.

Reply:
While this is technically not part of the public API, TESStesselator is part of the public interface and it's possible for clients to access its fields. Not sure it's worth doing now, if I'm going to do that sort of cleanup I'd like to leave it for a separate change.